data/json/items/armor/ballistic_armor.json

```diff
@@ -144,14 +152,15 @@
     "category": "armor",
     "name": { "str": "steel ballistic plate" },
     "repairs_like": "hsurvivor_jumpsuit",
-    "//": "based on 3a plates. Discuss high temp quenching so gonna assume tempered steel",
-    "//2": "link like: https://www.shindn.com/products/nij-iiia?variant=35614363582627&currency=USD&utm_medium=product_sync&utm_source=google&utm_content=sag_organic&utm_campaign=sag_organic&utm_campaign=gs-2019-08-03&utm_source=google&utm_medium=smart_campaign&gclid=Cj0KCQjw5-WRBhCKARIsAAId9FlGQAManRGZ0qpV9AIkD23FZfpm4aPDI4PHE9qPktZ2bdQY2NmcIVkaAs2pEALw_wcB",
-    "description": "A concave ballistic plate made of hard steel, designed to stop small-caliber rounds.  \"NIJ III A PROTECTION\" is etched into the back of the plate.  This is intended to be inserted into a ballistic vest, and can't be worn on its own.",
-    "weight": "1400 g",
-    "//3": "30cm x 25cm x .35cm",
-    "volume": "262 ml",
+    "//": "previous entry used some weird chinesium plate, added a known brand instead",
```

Comment:
The reason these used "chinesium" plates was they were a single material and simple. 

If you propagated your new measurements to the armor definition bellow it would be thickness: 10 of steel which is 100 bash resist, 100 cut resist, and 60 bullet resistance which believe it or not, doesn't make any sense. Now mind you that's because you used a ceramic body armor reference for steel plates.

Reply:
Thanks for catching this mistake, I didn't double check the material.

Doing some research, I cannot find **ANY** manufacturer that produces a level IIIA STEEL plate.  

Most things I see in that range are polyethylene with a few ceramic plates thrown in the mix.  Should this one be converted to a ceramic plate?  I don't think we have UHWMP as a material in the game.

Reply:
For the time being, I just reverted it.  It would make sense as a UHWMPE plate but that material isn't in the game yet.